Makefile

```diff
@@ -44,16 +43,16 @@ clean:
 	rm -rf lib test/test
 
 lint:
-	$(DEPS_BIN_DIR)/eslint src
+	npm run lint
```

Comment:
If we simply run the `npm` scripts from the `Makefile`, we should be consistent about it. So I think we should do the same for the `dev` and `watch` targets.